toDoList/src/index.js

```diff
@@ -0,0 +1,72 @@
+import './style.css';
+
+const tasks = [
+  {
+    description: 'Finish project',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Clean the house',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Take Walk',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Hangout with friends',
+    completed: true,
+    index: 3,
+  },
+  {
+    description: 'Rest for 8 hours',
+    completed: false,
+    index: 4,
+  },
+];
+
+// Function to iterate over tasks array and create HTML list items
+function showTasks() {
+  const todoList = document.getElementById('listContainer');
+  const items = document.createElement('ul');
+  items.classList.add('items');
+  todoList.innerHTML = '';
+  const taskTitle = document.createElement('p');
+  taskTitle.innerText = "Today's To Do";
+  taskTitle.classList.add('taskTitle');
+  todoList.appendChild(taskTitle);
+  const addTask = document.createElement('input');
+  addTask.setAttribute('placeholder', 'Add Task');
+  addTask.classList.add('addTask');
+  todoList.appendChild(addTask);
+
+  const removeTask = document.createElement('button');
+  removeTask.innerText = 'Clear Complete Tasks';
+  removeTask.classList.add('removeTask');
+
+  tasks.forEach((task) => {
+    const listItem = document.createElement('label');
+    const listInput = document.createElement('input');
+    const x = document.createElement('BR');
+    const hr = document.createElement('HR');
+    listInput.type = 'radio';
```

Comment:
Nice design 🎨 but in order to stick to the design and make it behave like a real list (ability to check and uncheck tasks ☑️ ) you should consider using a `checkbox` instead of a `radio`, the `radio` doesn't allow you to uncheck a selection but the `checkbox` does, plus it fits the design required. 👇🏽 

## Required task item design
![Screenshot 2023-02-27 at 20 30 55](https://user-images.githubusercontent.com/45739963/221729365-ed90835b-2928-4629-b092-34657a93d62c.png)

## What you have
![Screenshot 2023-02-27 at 20 32 05](https://user-images.githubusercontent.com/45739963/221729537-b82af419-d452-4496-bddb-0eb15bab8acd.png)

You can fix this by replacing what you have by this:
```
listInput.type = 'checkbox';
```